packages/manager/modules/pci/src/projects/project/instances/add/add.html

```diff
@@ -305,6 +324,7 @@ <h1 data-translate="pci_projects_project_instances_add_title"></h1>
         </div>
 
         <div
+            data-ng-show="false"
```

Comment:
What's the utility ?
```suggestion
```

Reply:
Its been removed in next PR.  https://github.com/ovh/manager/pull/10523